includes/auto_loaders/config.paypalwpp_incontext.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Autoloader array for PayPal Express In Context functionality. Makes sure that PayPalWPP In Context is instantiated at the
+ * right point of the Zen Cart initsystem.
+ * 
+ * @package     paypalwpp_incontext
+ * @author      mc12345678 
+ * @copyright   Copyright 2008-2016 mc12345678
+ * @copyright   Copyright 2003-2007 Zen Cart Development Team
+ * @copyright   Portions Copyright 2003 osCommerce
+ * @link        http://www.zen-cart.com/
+ * @license     http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version     $Id: config.paypalwpp_incontext.php xxxx 2016-08-20 11:15:10Z mc12345678 $
+ *
+ * Stock by Attributes 1.5.4
```

Comment:
Considering that some of the files could be removed (are needed only to support paypal_incontext)  and the overall functionality would remain (payment possible through PayPal), I plan on modifying to paypal_incontext for newly added files.

Will address other inconsistencies with the docblocks as well.  Expect to be able to provide updated (and corrected) within the next 14 hours. I encourage not giving up on the review and continuing to identify other issues no matter how "small".